gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/BuildController.groovy

```diff
@@ -37,28 +39,33 @@ class BuildController {
   @Autowired
   BuildService buildService
 
+  @ApiOperation(value = "Get build masters", response = List.class)
   @RequestMapping(value = "v2/builds", method = RequestMethod.GET)
   List<String> getBuildMasters() {
     buildService.getBuildMasters()
   }
 
+  @ApiOperation(value = "Get jobs for build master", response = List.class)
   @RequestMapping(value = "/v2/builds/{buildMaster}/jobs", method = RequestMethod.GET)
   List<String> getJobsForBuildMaster(@PathVariable("buildMaster") String buildMaster) {
     buildService.getJobsForBuildMaster(buildMaster)
   }
 
+  @ApiOperation(value = "Get job config", response = HashMap.class)
```

Comment:
Any reason why all the `List` endpoints use `List.class` but the `Map` endpoints use `HashMap.class`?